src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(num) {
+    if (num === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      sum += num;
+
+      return adder;
+    }
```

Comment:
No need to use **else** here:
```suggestion
    if (num === undefined) {
      const result = sum;

      sum = 0;

      return result;
    } 
      sum += num;

      return adder;
    
```

Reply:
okay :). Already fixed!